src/core/tests/frontend/shared/include/convert_model.hpp

```diff
@@ -6,7 +6,7 @@
 
 #include <gtest/gtest.h>
 
-#include <manager.hpp>
+#include <openvino/frontend/common/frontend_manager.hpp>
```

Comment:
On the install tree I see:
```sh
common
manager.hpp
onnx_frontend
onnx_import
paddlepaddle_frontend
```
So, we can see:
- FEM header file is named `manager.hpp`
- It's located in root folder with FEs

CC @mvafin @slyalin @nosovmik I suppose this is what we agreed when reviewed renaming FEM => FE common

Reply:
Yes we agreed to rename to manager.hpp  before. However I'd prefer to have in code `#include <openvino/frontend/manager.hpp>` instead of current's `#include <manager.hpp>`

Reply:
renamed to manager.hpp